dev-python/logbook/logbook-1.5.3.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_IN_SOURCE_BUILD=1
```

Comment:
Did you verify whether this is necessary?

Reply:
It builds without it and the tests are passing